src/main/java/com/rocketden/main/dto/room/RoomMapper.java

```diff
@@ -9,12 +9,11 @@ protected RoomMapper() {}
 
     private static final ModelMapper mapper = new ModelMapper();
 
-    public static CreateRoomResponse entityToCreateResponse(Room entity) {
-        return mapper.map(entity, CreateRoomResponse.class);
-    }
-
-    public static JoinRoomResponse entityToJoinResponse(Room entity) {
-        return mapper.map(entity, JoinRoomResponse.class);
+    public static RoomDto toDto(Room entity) {
```

Comment:
Question: Could we have just one "Mapper" class for all by passing in generics? Would that even be helpful, or is that unnecessary?

Reply:
I think we could (especially at this stage since we don't have that many), but I think the reason for keeping the separate mappers would be for flexibility as things grow more complex and the black-box ModelMapper isn't able to do all of it for us (such as if variable names don't match, or if conversion requires other info such as game data or something)

Reply:
Ok I think that makes sense -- yeah I found that implementing a [generic version](https://github.com/modelmapper/modelmapper/issues/187) actually seems complicated, so this is OK.